pilot/pkg/networking/core/v1alpha3/route/route.go

```diff
@@ -600,6 +601,36 @@ func BuildDefaultHTTPRoute(node *model.Proxy, clusterName string, operation stri
 	return defaultRoute
 }
 
+// translatePercentToFractionalPercent translates an v1alpha3 Percent instance
+// to an envoy.type.FractionalPercent instance.
+func translatePercentToFractionalPercent(p *networking.Percent) *xdstype.FractionalPercent {
+	fp := xdstype.FractionalPercent{}
+	switch {
+	case p.Value >= 1.0:
+		fp.Numerator = uint32(p.Value)
```

Comment:
This is not the right way. How do we support 1.25%? 

You need to multiply the value by 1000000 and  use it as the numerator, denominator being million.

Reply:
Multiply by 1000000 or 10000?